packages/server/graphql/mutations/updatePokerScope.ts

```diff
@@ -166,10 +166,15 @@ const updatePokerScope = {
       // MUTATIVE
       newDiscussions.push(...discussions)
       stages.push(...newStages)
-      newStageIds = newStages.map(({id}) => id)
```

Comment:
We were looping through all the updates in `additiveUpdatesWithTaskIds` and then using `dimensions` to get the `newStages`. However, we would then overwrite `newStageIds` instead of appending them. 

This meant that only one of two (depending on the number of dimensions) `newStageIds` would be returned from the mutation.